### ISSUE TITLE
Proposal: Transferring pypi ownership of package name "salve"

### ISSUE THREAD

**Issue by Moosems** [sirosen/SALVE#12]
Hello @sirosen!

My name is Moosems, and I am the maintainer and developer of the package [Salve IPC](https://github.com/Moosems/salve) which goes by "Salve" for short. I initially wanted to use Salve for my package name when I first dreamed it up but since this repository had taken it, I decided to tack on "IPC" at the end because it is in fact an IPC library (hence the "salve" which translates to "hello" from Italian). Unfortunately, this repository has not been maintained for 9 years now (by the way, having looked at the website and snooping around a bit, its actually pretty cool and a very large project). However, it remains that this is not exactly a maintained library anymore. Since my library uses the name "Salve" for short, I would really love to be able to use the name on Pypi and thought I'd ask if you would be willing to transfer the name to me. I understand if this is not something you want or are willing to do, and I won't hold anything against you should you choose to deny my proposal, but I thought I'd at least ask you if for nothing more than to ask. If you are interested, the idealized plan for transferring ownership is you can either change the project name on pypi or delete it so the space opens up and I simply change the project name from my pypi account for Salve IPC to Salve. Ping me sometime if you would like to talk! Hope this finds you on a wonderful day! Thank you for your time and understanding,

 - Moosems

**Comment by Moosems** [comment]
[This](https://discuss.python.org/t/transferring-a-pypi-name-to-someone-else/7196) is also another possibility

**Comment by sirosen** [comment]
I'm happy to relinquish the package name; it's been a very long time since I abandoned this project and I doubt anyone is using it (frankly, I hope not).

For a dead project like this, with no user-base, I think there's little point in delaying. I've deleted the project from pypi and the name should now be available. I might come back to this repo to put some kind of final archive notice on it.

Cheers and happy hacking!
